start_admin_sshd.sh

```diff
@@ -47,9 +35,23 @@ get_user_data_keys() {
     ( IFS=$'\n'; echo "${valid_keys[*]}" )
 }
 
+declare -r PERSISTENT_STORAGE_BASE_DIR="/.bottlerocket/host-containers/current"
+declare -r SSH_HOST_KEY_DIR="${PERSISTENT_STORAGE_BASE_DIR}/etc/ssh"
+declare -r USER_DATA="${PERSISTENT_STORAGE_BASE_DIR}/user-data"
+
+declare -r LOCAL_USER="ec2-user"
+declare -r USER_SSH_DIR="/home/${LOCAL_USER}/.ssh"
+
+# This is a counter used to verify at least
+# one of the methods below is available
+available_auth_methods=0
```

Comment:
Please declare this as an integer to avoid introducing bugs in later revisions: 

```suggestion
declare -i available_auth_methods=0
```

Example of why this is important:

```bash
#!/usr/bin/env bash
unset fooint foostr
declare -i fooint=0
foostr=0

fooint+=1
foostr+=1

echo "fooint: $fooint, foostr: $foostr"

#stdout: fooint: 1, foostr: 01
```